site/content/docs/main/custom-plugins.md

```diff
@@ -54,10 +54,14 @@ You will need to give your plugin(s) the full name when registering them by call
 Velero supports the following kinds of plugins:
 
 - **Object Store** - persists and retrieves backups, backup logs and restore logs
-- **Volume Snapshotter** - creates volume snapshots (during backup) and restores volumes from snapshots (during restore)
+- **Item Snapshotter** - creates snapshots for Kubernetes objects during backup and restores the object from snapshots during restore.  ItemSnapshotters
+  are typically used with the [Astrolabe](https://github.com/vmware-tanzu/astrolabe) framework.
```

Comment:
Mentioned during community meeting today. Any details on this integration?

Reply:
This PR is the APIs.  More to come after the APIs are in.